### DISCUSSION TITLE
How do I save my project settings/template after loading source file?

### DISCUSSION THREAD

**Discussion by alvinek** [staxrip/staxrip#1567] [Q&A]
So I'm new to this app and I'm already loving it. I was messing around with settings for my drone videos, encoded one several times using different settings for NVENC and I got what I wanted. However, I have no idea how to save my project settings to use them for another video. Tried clicking File -> Save project as template but I keep getting an warning that I cannot create a template after a source file was opened. I don't see any option to unload source file. 

I don't know what is the best workflow for this app, I thought I could save settings and add rest of my videos with saved settings, but this seems to be not possible or I'm doing something wrong.

**Comment by Dendraspis** [comment]
Hello and welcome.

It's not possible to do it that way. You have to make your settings without a source file, means from another template like directly after app start, then you can save it as a new template.

Via ` File » Project Templates` you can select already saved templates.

**Comment by JonathonSunshine** [comment]
You can save the "video filters" ,  "audio encoding" and "video encoding" settings separately, then load them on top of a blank template and save that. But you can't save the project options that way.
